packages/deployment-service/.env.dist

```diff
@@ -0,0 +1,4 @@
+MYSQL_ROOT_PASSWORD=root
```

Comment:
Erm, I think this should be gitignored. And probably best to change the PW

Reply:
This is fine, it's a dist so the idea is to copy this from `.env.dist` to `.env` and use `.env` for dev